From: Otto Kekäläinen <otto@debian.org>
Date: Tue Aug 20 21:56:25 2024 -0700
Subject: Fix man page environment.5 link to man5/pam_env.conf.5

Fix Lintian error:

    E: libpam-modules-bin: bad-so-link-within-manual-page [usr/share/man/man5/environment.5.gz]

--- a/modules/pam_env/environment.5
+++ b/modules/pam_env/environment.5
@@ -1 +1 @@
-.so pam_env.conf.5
+.so man5/pam_env.conf.5
